App.js

```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 
 export default function App() {
+  const URL = 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest';
+  useEffect(() => {
+    fetch(URL, {
+      headers: {
+        'X-CMC_PRO_API_KEY': '07cff7a2-6b5c-4328-86d1-77fceed0f357'
```

Comment:
You cannot expose the API key like that
Create a `.env` and add it to the `.gitignore` file. In the root directory and add the api key like this:
`API_KEY=07cff7a2-6b5c-4328-86d1-77fceed0f357`
then do `process.env.API_KEY` where you want to use it

Reply:
Alright thanks for the feedback